Examples/Tests/Macroscopic_Maxwell/inputs_3d_noMs

```diff
@@ -1,15 +1,14 @@
-# This is a E+B (use USE_LLG=FALSE, ie., no M field) simulation of a
+# This is a E+H (use USE_LLG=FALSE) simulation of a
 # plane wave in a periodic domain
-
 ################################
 ####### GENERAL PARAMETERS ######
 #################################
-max_step = 500
+max_step = 1000
 amr.n_cell = 16 16 128
 amr.max_grid_size = 32
 amr.blocking_factor = 16
 geometry.coord_sys   = 0
-geometry.is_periodic = 1  1  0
+geometry.is_periodic = 1  1  1
```

Comment:
was there a reason for this to be changed to periodic?

Reply:
oh no, this is just the residual in the test. I'll change it back

Reply:
got it!